### PR TITLE
redesigned popup btn

### PR DESCRIPTION
**Pull request by cfalucho**

### CODE REVIEW

File: components/EventCard/EventCard.component.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const EventCard = (props) => {
                     <Image style={styles.imagePopUp} resizeMode="contain" source={props.source} />
                     <Text style={styles.datePopUp}> {props.date} </Text>
                     <Text> {props.link} </Text>
-                    <TouchableOpacity style={styles.btnPopUp}>
+                    <TouchableOpacity style={styles.btnPopUpRSVP}>
                         <Text style={styles.btnText} 
                         onPress={
                             () => {
```
Comment (exelarios) [inline components/EventCard/EventCard.component.js:37]: Use the custom Button called "MainButton" in our components, it's basically the same thing you're doing but it's just group into a component.

Comment (cfalucho) [reply components/EventCard/EventCard.component.js:37]: Oh okay, my bad. Did not see that. I will make that adjustment. Thanks!

Comment (cfalucho) [reply components/EventCard/EventCard.component.js:37]: Wait, the "MainButton" adjusts every other button. I was focusing more on the buttons when the card pop-ups. Unless I'm mistaking something.

Comment (exelarios) [reply components/EventCard/EventCard.component.js:37]: You could add custom styling to the MainButton component. It's fine, I can fix it. lol

**Review by exelarios — APPROVED** [review]

**Pull request merged**